src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -132,6 +133,13 @@ pub struct ChronicleUnstructuredConfig {
     #[configurable(metadata(docs::examples = "production"))]
     pub namespace: Option<String>,
 
+    /// A set of labels that are attached to each batch of events.
+    ///
+    /// Both keys and values are not templateable.
```

Comment:
Is there a reason why these shouldn't be templateable?

Reply:
I'm new to Vector, keeping it easy for now. This MR is a Draft.

Reply:
Yes, partitioning is where it gets more complicated. You would need to implement the [`Partitioner`](https://github.com/vectordotdev/vector/blob/master/lib/vector-core/src/partition.rs#L7) trait that extracts distinct data from the event to determine which batch it should go in. The [`loki`](https://github.com/vectordotdev/vector/blob/master/src/sinks/loki/sink.rs#L456) sink has an example of this. Partitioning on a `HashMap` of templates would be a bit more complex - but certainly possible.

Only if you are up for the challenge though! It's not essential to get this PR merged since non templateable fields are still an improvement over the current behaviour. It can be revisited based on user demand.

I would probably like Namespace to be templateable at some point too. The author of that PR didn't need it to be, so we are waiting for someone who has the time and need to change it.

Reply:
Moving forward with non templateable fields for now.